openedx/core/djangoapps/credentials/signals.py

```diff
@@ -74,7 +74,7 @@ def send_grade_if_interesting(user, course_run_key, mode, status, letter_grade,
         return
 
     # Avoid scheduling new tasks if learner records are disabled for this site.
-    if not helpers.get_value_for_org(course_run_key.org, 'ENABLE_LEARNER_RECORDS', True):
+    if not is_learner_records_enabled():
```

Comment:
Either I missed this or it changed, but this should use is_learner_records_enabled_for_org.

Reply:
Good catch!